src/NuGetRepack/Common/NuGetVersionUpdater.cs

```diff
@@ -250,6 +270,25 @@ private static void LoadPackages(IEnumerable<string> packagePaths, Dictionary<st
             }
         }
 
+        private static bool IsDotnetTool(string nuspecXmlns, XElement metadata)
+        {
+            var packageTypesElement = metadata.Element(XName.Get("packageTypes", nuspecXmlns));
+            if (packageTypesElement != null)
+            {
+                foreach (var packageType in packageTypesElement.Elements(XName.Get("packageType", nuspecXmlns)) ?? Array.Empty<XElement>())
+                {
+                    var name = packageType.Attribute("name");
+
+                    if (name != null && name.Value != null && string.Equals(name.Value, "DotnetTool", StringComparison.OrdinalIgnoreCase))
```

Comment:
>name != null && name.Value != null && string.Equals(name.Value [](start = 24, length = 62)

Nit: Can be simplified to `string.Equals(name?.Value, "DotnetTool", StringComparison.OrdinalIgnoreCase)`.

Reply:
Used suggested